static/app/components/replays/replayView.tsx

```diff
@@ -41,7 +41,7 @@ function ReplayView({toggleFullscreen}: Props) {
               </Button>
             ) : null}
           </ContextContainer>
-          {replay?.hasProcessingErrors() ? (
+          {!isFetching && replay?.hasProcessingErrors() ? (
```

Comment:
This condition wasn't copied over properly from [static/app/views/replays/details.tsx](https://github.com/getsentry/sentry/pull/61388/files#diff-db656f76ee6f8dda9a3659e2531ce0d485de0b9bf859313ae4967facb0889566) in #61388